src/loader.js

```diff
@@ -146,15 +147,99 @@
   };
 
   // 4. Registry Objects
+  // For now, registry objects are a work in progress that don't fully integrate into the rest of the code base
 
   // 4.1.1
+  function createRegistry() {
+    var registry = new RegistryPrototype();
+    registry._registry = {};
```

Comment:
Unless you were talking about line 155 specifically, where `registry._registry` is being created?  That line was intended to implement steps 2-3 of https://whatwg.github.io/loader/#registry-CreateRegistry. The reason for using a `_registry` property was to follow the pattern I had seen elsewhere about underscore-prefixed properties that are "internal" (see [example](https://github.com/ModuleLoader/es6-module-loader/blob/1.0/src/loader.js#L13)). 

However, now seeing your other comment about the above diff, it looks like we're planning on deviating a bit from the spec in the name of simplicity. So in that case, I can just put all of this into the `Registry` constructor.